### DISCUSSION TITLE
Question about awesome yourls

### DISCUSSION THREAD

**Discussion by SophiaAtkinson** [YOURLS/YOURLS#3815] [Q&A] [answered]
Does the plugin have to be on GitHub? I use my own git server, and that's where all of my code goes, it's annoying to have to maintain code in 2 places. 
Am I allowed to link to my instance?

Its git.oldgate.org ffr

Sorry if this has been asked before

**Comment by LeoColomb** [comment, marked as answer]
> Am I allowed to link to my instance?

Absolutely, as soon as the source code can be cloned/duplicated and is distributed with an open-source license, all is good!

**Reply by SophiaAtkinson** [reply]
Great! Thank you for the help, its just standard Gitea, so works very similar to GitLab
